src/api/doiRequestApi.ts

```diff
@@ -0,0 +1,17 @@
+import { CancelToken } from 'axios';
+import { authenticatedApiRequest } from './apiRequest';
+
+enum DoiRequestApiPaths {
+  DOI_REQUEST = '/doi-request',
+}
+
+export const createDoiRequest = async (publicationId: string, message?: string, cancelToken?: CancelToken) =>
+  await authenticatedApiRequest({
+    url: DoiRequestApiPaths.DOI_REQUEST,
+    method: 'POST',
+    data: {
+      publicationId,
+      message,
+    },
+    cancelToken,
+  });
```

Comment:
hvis du har en egen fil for doiRequests, så synes jeg `getDoiRequests` som ligger i publicationApi også burde flyttes hit

Reply:
ja, begynte faktisk på det, men tenkte det var bedre å bare droppe det nå og ta det i NP-1047:Use endpoint for fetching doi requests, siden det blir litt omskriving der også

Reply:
ah oki 👍